### PR TITLE
Ability to translate for a pair of strings

### PR DESCRIPTION
**Pull request by tltneon**
image

### CODE REVIEW

File: src/client/components/common/PurgeWarning.vue [hunk 1]
```diff
@@ -36,5 +36,14 @@ export default Vue.extend({
       return '' + date.getFullYear() + '-' + pad(date.getMonth()+1) + '-' + pad(date.getDate()) + ' ' + pad(date.getHours()) + ':' + pad(date.getMinutes());
     },
   },
-});
+  methods: {
+    getWarningText(isHours: Boolean) {
+      if (isHours) {
+        return translateTextWithParams('Warning: This game will be purged in approximately ${0} hours.', [Math.floor(this.hoursLeft).toString()]);
```
Comment (kberg) [inline src/client/components/common/PurgeWarning.vue:42]: Huh, we can't do this with v-i18n. OK./

**Review by kberg — APPROVED** [review]
Thanks for this. This is a nice change.

**Comment by kberg** [comment]
... except for the build failures. :D

**Comment by tltneon** [comment]
Successfully builds for my but throwing errors by linting here, meh

**Comment by kberg** [comment]
> Successfully builds for my but throwing errors by linting here, meh

I'll look.

**Comment by kberg** [comment]
I have sent you a fix by pull request.

**Pull request merged**